spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/KafkaBinderTests.java

```diff
@@ -107,6 +116,19 @@ public void init() {
 		}
 	}
 
+	@After
+	public void cleanup() {
+		try {
+			Seq<String> topics = ZkUtils.getAllTopics(kafkaTestSupport.getZkClient());
+			for (String topic: scala.collection.JavaConversions.seqAsJavaList(topics)) {
+				AdminUtils.deleteTopic(kafkaTestSupport.getZkClient(), topic);
+			}
+		}
+		catch (Exception e) {
+			log.error(e.getMessage());
```

Comment:
Looks like in the current config of the tests it isn't possible to use the external broker as 'true' is harcoded.  (I changed it by hand in the code for my testing).  So either we should support using an external broker or now.  also, even w/ the embedded brokers, if the tests run in parallel then deleting all of them would run into some race conditions.  Might not be worth the effort at this point... it looks like all the destinations created are unique, so if we just for embedded, get rid of the option to use external and we don't need to cleanup anything.